### PR TITLE
18 add report screenshots

### PR DESCRIPTION
**Pull request by leonardcser**
Below are the changes mentioned in the issue #18. 

The ´media´ folder is placed under ´the-guide´ folder. It would contain all the specific media for the guide. The images use a relative path because it seems like absolute paths only work if the media folder is at the same level of the markdown file.

For the custom style for the images (21f0e07e34ba8444178f312594c01310f817e9d1), the value `.2rem` is the same as for the info/warning boxes.

### CODE REVIEW

File: docs/the-guide/media/plots_diff.png
Comment (ludelafo) [inline docs/the-guide/media/plots_diff.png:1]: Have you compressed your images with a software such as [Caesium Image Compressor](https://caesium.app/) to save all available space? You can compress the files "losslessly" and still save a few KBs.

**Review by ludelafo — CHANGES_REQUESTED** [review]
I would rather use the "official" `assets/images` directory such as on the official Material for MkDocs repository: https://github.com/squidfunk/mkdocs-material/tree/master/docs/assets/images (usage here: https://github.com/squidfunk/mkdocs-material/blob/master/docs/reference/images.md?plain=1#L177-L178) instead of the `media` directory in the `the-guide` directory as we might want to add future images that aren't part of The guide. What do you think?

File: docs/stylesheets/extra.css [hunk 1]
```diff
@@ -9,3 +9,7 @@ code {
 [data-md-color-scheme="slate"] {
 	--md-code-hl-color: var(--md-accent-fg-color--transparent);
 }
+
+article img {
+	border-radius: .2rem;
+}
```
[the remark this answers is not in the document]
Comment (leonardcser) [reply docs/stylesheets/extra.css:15]: The difference is very subtle. It slightly rounds the corners of the images inside of `article` tags, so the main content of the docs. This is to keep style consistency with the rounding of info/warning boxes and code blocks.

[the remark this answers is not in the document]
Comment (ludelafo) [reply docs/stylesheets/extra.css:15]: Ok! Good point, thank you for the explanation.

**Review by ludelafo — APPROVED** [review]
All good for me!

**Pull request merged**